packages/client/components/AddTeamDialog.tsx

```diff
@@ -133,7 +111,7 @@ const AddTeamDialog = (props: Props) => {
     )
   }
 
-  const isValid = selectedUsers.length && teamName.trim()
+  const isValid = teamName.trim().length > 0
```

Comment:
This fixes a bug in prod where you can only create a team if you invite team members. You should be able to create a team just by adding the team name.